crates/rspack_binding_options/src/options/raw_optimization.rs

```diff
@@ -36,7 +36,12 @@ impl RawOptionsApply for RawOptimizationOptions {
     self,
     plugins: &mut Vec<Box<dyn rspack_core::Plugin>>,
   ) -> Result<Self::Options, rspack_error::Error> {
+    let mut delimiter = "~".to_string();
```

Comment:
default logic should move to js side, checkout https://github.com/webpack/webpack/blob/87660921808566ef3b8796f8df61bd79fc026108/lib/config/defaults.js#L1368

Reply:
done